code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -544,8 +550,8 @@ components:
               accessPoints:
                 $ref: '#/components/schemas/AccessEndpoint'
           minItems: 1
-    EdgeCloudNodeName:
-      description: Edge Cloud Node Name - an identifier for an edge cloud node in the operator domain 
+    EdgeCloudZone:
```

Comment:
'String' risks conflict across providers . To ensure a unique value here, we should add another property `edgeCloudProvider` which represents the edge cloud provider name (e.g. the operator or hyperscaler) . Together the provider and edge cloud zone name tuple will be unique.

Reply:
Hi Kevsy,

We agree on adding Provider parameter but we left naming agreement between SED and EdgeCloud_LCM for the next release.